src/documentation/components/ThemeSwitcher.vue

```diff
@@ -80,24 +80,68 @@
 
 <script lang="ts">
 import { Themes } from '../models/models';
-import { THEMES } from '../constants/constants';
+import { THEMES, defaultCss, defaultDocsCss } from '../constants/constants';
 import { Component, Vue } from 'vue-property-decorator';
 
 declare const chi: any;
+interface AssetsToReplace {
+  type: 'css' | 'docsCss';
+  id: string;
+}
 
-@Component({})
+@Component({
+  head() {
```

Comment:
I'm getting a "No overload matches this call." error in head().

Reply:
Do you mean in IDE?
Standard Vue extensions of visual studio code don't support Nuxtjs methods like this one and mark it as error.

Here is Nuxt.js documentation https://nuxtjs.org/docs/components-glossary/head/